### DISCUSSION TITLE
Add a conversion macro can convert intervals to seconds

### DISCUSSION THREAD

**Discussion by WilliamGuozi** [grafana/grafana#52688] [Observability]
https://community.grafana.com/t/how-to-convert-intervals-to-seconds/11854

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
